nailgun/entities.py

```diff
@@ -3397,6 +3402,20 @@ def read(self, entity=None, attrs=None, ignore=None):
         ignore.add('discovery')
         return super(Subnet, self).read(entity, attrs, ignore)
 
+    def update_payload(self, fields=None):
+        """Wrap submitted data within an extra dict."""
+        return {u'subnet': super(Subnet, self).update_payload(fields)}
+
+    def update(self, fields=None):
+        """Fetch a complete set of attributes for this entity.
+
+        For more information, see `Bugzilla #1250922
+        <https://bugzilla.redhat.com/show_bug.cgi?id=1250922>`_.
```

Comment:
Is this method necessary? I can completely drop this `update` method and sucessfully update a subnet. Also, the linked-to bug is entitled "PUT /api/compute_resources/:id returns an incomplete set of attributes". If this method is necessary, it should link to a relevant bug.